vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -3,15 +3,14 @@ sidebarDepth: 2
 ---
 
 # Entando Hub
-## Overview
 
 An Entando Hub enables teams to share components across their organization and between Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7 App Builder.
```

Comment:
Hub 3.x uses the v5 format so it can no longer be installed in Entando 6.
```suggestion
An Entando Hub enables teams to share components across their organization and between Entando Applications. It can be installed in any Entando 7+ instance and includes API-level integration with the App Builder.
```